Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Configurations/TouchActionsSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Aquality.Selenium.Core.Configurations;
+
+namespace Aquality.Appium.Mobile.Configurations
+{
+    public class TouchActionsSettings : ITouchActionsSettings
+    {
+        private readonly ISettingsFile settingsFile;
+        private readonly string swipeSettingsPath = ".touchActions.swipe";
+
+        public TouchActionsSettings(ISettingsFile settingsFile)
+        {
+            this.settingsFile = settingsFile;
+        }
+
+        public int SwipeRetries => settingsFile.GetValue<int>($"{swipeSettingsPath}.retries");
+
+        public long SwipeDuration => settingsFile.GetValue<long>($"{swipeSettingsPath}.duration");
+
+        public float SwipeVerticalOffset => settingsFile.GetValue<float>($"{swipeSettingsPath}.verticalOffset");
```

Comment:
seems that we usually use double type for settings parameters.
Also we use it in java implementation https://github.com/aquality-automation/aquality-appium-mobile-java/pull/13/files#diff-7904949e13f61180f1d2a108ad08c00744395690df8f8edd3d61acd9ccb8cb7c
why to use floats here?

Reply:
@mialeska just reinsurance, cos if `horizontalOffset` is double there might be possible `OverflowException`.

Reply:
"verticalOffset": 0.2,
      "horizontalOffset": 0.5
      
     I don't get it, how could we get OverflowException for values 0 < value < 1 ?

Reply:
Both offset values are user configurable, which means they could be way more than 1 (though it makes no sense).
I agree that it will cause `OverflowException`, but on the other hand we don't need high precision that `double` provides. That's why I was thinking of `float`

Reply:
hmm, do we need any validation? e.g. we have it for TimeoutConfiguration in ctor, so miscunfigured solution won't run and we know it earlier